spec/fog/openstack/identity_v3_spec.rb

```diff
@@ -40,20 +40,18 @@
     # Allow us to ignore dev certificates on servers
     Excon.defaults[:ssl_verify_peer] = false if ENV['SSL_VERIFY_PEER'] == 'false'
 
-    unless example.metadata[:skip_preauthentication]
-      VCR.use_cassette('idv3') do
-        @id_v3 = Fog::Identity::OpenStack::V3.new(
-            :openstack_project_name => ENV['OS_PROJECT_NAME'] || 'admin',
-            :openstack_domain_name => ENV['OS_USER_DOMAIN_NAME'] || 'Default',
-            :openstack_api_key => ENV['OS_PASSWORD'] || 'password',
-            :openstack_username => ENV['OS_USERNAME'] || 'admin',
-            :openstack_region => ENV['OS_REGION_NAME'] || 'RegionOne',
-            :openstack_auth_url => "#{@os_auth_url}/auth/tokens") unless @id_v3
-      end
+    VCR.use_cassette('idv3') do
+      @id_v3 = Fog::Identity::OpenStack::V3.new(
+          :openstack_project_name => ENV['OS_PROJECT_NAME'] || 'admin',
+          :openstack_domain_name => ENV['OS_USER_DOMAIN_NAME'] || 'Default',
+          :openstack_api_key => ENV['OS_PASSWORD'] || 'password',
+          :openstack_username => ENV['OS_USERNAME'] || 'admin',
+          :openstack_region => ENV['OS_REGION_NAME'] || 'RegionOne',
+          :openstack_auth_url => "#{@os_auth_url}/auth/tokens") unless @id_v3
```

Comment:
Unrelated to your change, I often think /auth/tokens should automatically be appended to the auth_url so Fog would be more compatible with standard OpenStack env configs.